__tests__/hierarchy.ts

```diff
@@ -40,32 +40,20 @@ describe("Container to Contained", () => {
     expect(
       app.container.canConnect(app.container.contained.a, app.container.a)
     ).toBe(false);
-    expect(
-      app.container.canConnect(
-        app.container.contained.a,
```

Comment:
Leaving a marker for @axmmisaka to review.

Reply:
Ibidem - et seq.

Reply:
I made some changes related to this and left the description of them in this comment https://github.com/lf-lang/reactor-ts/pull/186#discussion_r1275695631.